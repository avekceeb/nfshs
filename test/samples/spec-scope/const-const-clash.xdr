const A = 1;
const B = 2;
const A = 3;