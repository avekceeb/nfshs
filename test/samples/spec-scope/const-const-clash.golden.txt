test/samples/spec-scope/const-const-clash.xdr:3:9:
Identifier "A" has been already declared at test/samples/spec-scope/const-const-clash.xdr:1:1
